src/css/addBook.module.css

```diff
@@ -0,0 +1,52 @@
+.container {
+  width: 100%;
+  display: flex;
```

Comment:
- [ ] As the previous reviewer mentioned you should not use  CSS bad practices[(article for support)](https://speckyboy.com/good-bad-css-practices/) like no repeating of the same rule. For example you have used `display: flex` 7 times. I would suggest you create a class example: **d-flex** and use it at all places that you need on your code

Reply:
Lol :laughing: I can't believe I'm getting lessons on CSS again. Thank you, man :+1: Let me implement that.